k8s/models/pod_disruption_budget.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+from __future__ import absolute_import
+
+import six
+
+from .common import ObjectMeta
+from ..base import Model
+from ..fields import Field, ListField
+
+
+class PodDisruptionBudgetMatchExpressions(Model):
```

Comment:
This class should be LabelSelectorRequirement to match the type in the Kubernetes API